src/passes/externalContractHandler/externalContractInterfaceInserter.ts

```diff
@@ -138,5 +145,38 @@ export function genContractInterface(
     });
   sourceUnit.appendChild(contractInterface);
   ast.registerChild(contractInterface, sourceUnit);
+  createImport(
```

Comment:
Wouldn't be better if we use a call to this 
https://github.com/NethermindEth/warp/blob/2a911fa0f38c15287f233030344228f939801832/src/cairoUtilFuncGen/base.ts#L51

and hardcode the `['super']` as a constant in the script where we have all import constants defined?

Reply:
well we can't use `this` here beacuse the class is of ASTMapper type, for `super` I can make a constant , good point